### ISSUE TITLE
add MASTER for the STREAM_QUALITY

### ISSUE THREAD

**Issue by kenshirothefist** [okonek/tidal-cli-client#54]
TIDAL hi-fi subscription also offers [MASTER](https://support.tidal.com/hc/en-us/articles/115000397069-TIDAL-MASTERS) for the STREAM_QUALITY (besides NORMAL, HIGH and HI-FI). Please add it as the config option. If MASTER quality is not available for a particular song/album, the app should change to Hi-Fi for a particular song/album. It would also be very nice that the client would display the current track info (length, quality mode, etc.). btw: keep up the good work, you app works great in ubuntu 18!

**Comment by fleyva1** [comment]
Master may not work for all laptop (to my understanding) and is dependent on the sound card (so it may get a bit convoluted on linux)?

**Comment by okonek** [comment]
Yes, the MASTER aka. MQA quality doesn't work the same as NORMAL, HIGH and HI-FI. I would really like to see what I can do about implementing that but now I have a very hard time in school, so I don't get much free time for this app. If you would like to try and implement it, I would be very happy to help you.

**Comment by shaun-ba** [comment]
As far as I see it this is the ONLY reason for this app, unless I'm missing something I have no idea why a CLI tool would be preferrable over the WebUI provided.

I stumbled upon this project because MASTER is only available in their desktop application, so yes, support for this would be amazing and I'm sure a major reason for all linux users to use it!
